integration-tests/oidc-code-flow/src/test/java/io/quarkus/it/keycloak/CodeFlowTest.java

```diff
@@ -499,6 +500,7 @@ public Boolean call() throws Exception {
     }
 
     @Test
+    @Disabled
```

Comment:
Can you make it `@Disabled("flaky")` and maybe add a link to the issue tracking it?